src/components/modules/nutrition/daily-intake/DailyIntakeMealHeader.vue

```diff
@@ -6,18 +6,22 @@
             <v-icon :class="expand ? 'rotated' : ''">expand_more</v-icon>
         </v-btn>
 
-        <v-btn
-            v-show="expand"
-            color="rgba(76, 175, 80, .85)"
-            dark
-            small
-            absolute
-            bottom
-            left
-            fab
-            class="food-search">
-            <v-icon>add</v-icon>
-        </v-btn>
+        <router-link
+          :to="{name: 'add-food', params: {meal: name.toLowerCase()}}"
+          tag="div">
+            <v-btn
+                v-show="expand"
+                color="rgba(76, 175, 80, .85)"
```

Comment:
I think adding zero (`0.85`) makes it a little bit more readable :)

Reply:
Done